docs/developer-guide/CLI Methods/generating-the-bundled-config-file.md

```diff
@@ -0,0 +1,87 @@
+# Generating the Bundled Configuration File
+
+[Running Basemaps Locally][running-basemaps-locally] \> [Configuring the Basemaps/server package][configuring-the-basemapsserver-package] \> [Providing a path to a bundled configuration JSON file][path-to-json-config-file] \> Generating the Configuration File
+
+This guide shows you how to generate the bundled configuration file using the **basemaps/cli** package.
+
+## Prerequisites
+
+### AWS Credentials
+
+You'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled configuration file required for running the **Basemaps/server** package.
+
+## Cloning the Repository
+
+### basemaps-config
+
+You must clone the **basemaps-config** repository to your machine.
+
+```bash
+# Clone using the web URL
+git clone https://github.com/linz/basemaps-config.git
+
+# Clone using a password-protected SSH key
+git clone git@github.com:linz/basemaps-config.git
+
+# Clone using GitHub's official CLI
+gh repo clone linz/basemaps-config
+```
+
+### Paths
+
+On your machine, consider the following paths:
+
+#### `BM_CONFIG_REPO`
+
+The path to the root folder of the **basemaps-config** repository on your machine. 
+
+```bash
+$BM_CONFIG_REPO = {path_to}/basemaps-config
+```
+
+#### `BM_CLI_BIN`
+
+The **basemaps/cli** package provides a **bundle** function to generate the bundled configuration file. The path to the **bin** folder of the **basemaps/cli** package is as follows:
+
+```bash
+$BM_CLI_BIN = $BM_REPO/packages/cli/bin
+```
+
+### Command
+
+Use the following command to generate the bundled configuration file:
+
+```bash
+node $BM_CLI_BIN/bmc.js bundle \
+    --config $BM_CONFIG_REPO/config \
+    --output $BM_REPO/config/config.bundle.json \
+    --cache s3://linz-basemaps-staging/basemaps-config/cache/
```

Comment:
This might only works for Raster map, vector map will require to bundle the asset location as well as following, assets are the fonts and glyphs that used for our vector map styles. 
`--assets s3://linz-basemaps/assets/assets-2irW8kbHduyqRxWyfa1xgk7t7sq1T3yeDLWqN6fR9T9H.tar.co`

@tawera-manaena How to bundle assets file is out of this scope, but for your reference to have a look, all the steps until (Bundle Assets Into Cotar) from https://github.com/linz/basemaps-config/blob/master/.github/workflows/build.yml

@blacha I just found that we don't have an `assets-latest.tar.co` uploaded into s3, do we require one now?

Reply:
I think we can leave the vector stuff out of scope for this PR, we could look at adding more improvements in a later pr.